src/components/shared/ManagedTree.css

```diff
@@ -15,7 +15,7 @@
 }
 
 .managed-tree .tree .node {
-  padding: 2px 3px 2px 3px;
```

Comment:
we need to keep this as the default for other managed trees:

we can add another style in sources.css

```css
.sources-list .managed-tree .tree .node {
  padding: 0px 0px 0px 3px;
}
```